Makefile

```diff
@@ -7,7 +7,7 @@ SHAREDIR ?= ${PREFIX}/share
 PYTHON ?= $(shell command -v python3 python|head -n1)
 DESTDIR ?= /
 PATH := $(PATH):$(HOME)/.local/bin
-MYPIP := pipx
+MYPIP ?= uv
```

Comment:
Should this be `uv pip`?

Reply:
I tried that, and it didn't pass, so I went back to just pip.